src/components/CavaticaProjectList/CavaticaProjectItem.js

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import {Link} from 'react-router-dom';
+import {List, Icon, Button, Popup, Divider} from 'semantic-ui-react';
+import TimeAgo from 'react-timeago';
+
+const CavaticaProjectItem = ({projectNode, unlinkProject, studyId}) => {
+  return (
+    <List.Item>
+      <Icon
+        name={
+          projectNode.projectType === 'DEL'
+            ? 'paper plane outline'
+            : 'sliders horizontal'
+        }
+      />
+      <List.Content>
+        {unlinkProject && studyId ? (
+          <List.Content floated="right">
+            <Popup
+              trigger={
+                <Button
+                  size="mini"
+                  basic
+                  negative
+                  onClick={e => e.stopPropagation()}
+                >
+                  UNLINK PROJECT
+                </Button>
+              }
+              header="Are you sure?"
+              content={
+                <>
+                  The project will remain in Cavatica even you unlink it from
```

Comment:
Maybe better phrased as "This will unlink the project from its study. It may always be linked back later."